ODIN_II/SRC/netlist_utils.cpp

```diff
@@ -239,20 +239,31 @@ npin_t* free_npin(npin_t* to_free) {
 	return (npin_t*)vtr::free(to_free);
 }
 
+/*-------------------------------------------------------------------------
+ * (function: copy_npin)
+ * 	Copies a pin
```

Comment:
specify in the comment that it should not be used but the parent function copy_input_npin copy_output_npin are the right one to us depending on the pin type